terraform.tfvars.example

```diff
@@ -5,35 +5,35 @@ cluster_id = "example-cluster"
 cluster_domain = "example-cluster.devcluster.openshift.com"
 
 // Base domain from which the cluster domain is a subdomain.
-base_domain = "devcluster.openshift.com"
+base_domain = "vmc.devcluster.openshift.com"
 
-// Name of the vSphere server. The dev cluster is on "vcsa.vmware.devcluster.openshift.com".
-vsphere_server = "vcsa.vmware.devcluster.openshift.com"
+// Name of the vSphere server. The dev cluster is on "vcsa.vmware.devcluster.openshift.com". //TODO
+vsphere_server = "vcsa.vmware.devcluster.openshift.com" //TODO
 
 // User on the vSphere server.
 vsphere_user = "YOUR_USER"
 
 // Password of the user on the vSphere server.
 vsphere_password = "YOUR_PASSWORD"
 
-// Name of the vSphere cluster. The dev cluster is "devel".
-vsphere_cluster = "devel"
+// Name of the vSphere cluster. The devqe cluster is "/DEVQEdatacenter/host/DEVQEcluster".
+vsphere_cluster = "/DEVQEdatacenter/host/DEVQEcluster"
 
-// Name of the vSphere data center. The dev cluster is "dc1".
-vsphere_datacenter = "dc1"
+// Name of the vSphere data center. The dev cluster is "DEVQEdatacenter".
+vsphere_datacenter = "DEVQEdatacenter"
 
-// Name of the vSphere data store to use for the VMs. The dev cluster uses "nvme-ds1".
-vsphere_datastore = "nvme-ds1"
+// Name of the vSphere data store to use for the VMs. The dev cluster uses "vsanDatastore".
+vsphere_datastore = "vsanDatastore"
 
 // Name of the vSphere portgroup used by the VMs.
-vm_network="ci-segment-15"
+vm_network="devqe-segment-xxx"
 
-// Name of the VM template to clone to create VMs for the cluster. The dev cluster has a template named "rhcos-latest".
-vm_template = "rhcos-latest"
+// Name of the VM template to clone to create VMs for the cluster. The dev cluster has a template named "rhcos-latest". //TODO
+vm_template = "rhcos-latest" //TODO
```

Comment:
The template value would match the name of the template that is uploaded to vCenter.